<launch>

<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
  output="screen" args="joint_state_controller  joint_right_effort_controller joint_left_effort_controller"/>

</launch>
